### PR TITLE
remove binstar references in strings printed out - anaconda instead

### PR DESCRIPTION
**Pull request by PeterDSteinberg**
Get rid of mentions of binstar in help printed out.  binstar -> anaconda

### CODE REVIEW

**Comment by PeterDSteinberg** [comment]
Fixes #213

File: binstar_build_client/build_commands/submit.py [hunk 1]
```diff
@@ -95,17 +95,17 @@ def submit_build(binstar, args):
...
         log.info('Build not submitted (dry-run)')
 
-def print_build_results(args, build):
+def print_build_results(args, build, binstar):
 
     log.info('')
     build_result_url = build.get('url')
     if not build_result_url:
-        build_result_url = 'http://anaconda.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build['build_no'])
+        build_result_url = 'http://%s/%s/%s/builds/matrix/%s' % (binstar.domain, args.package.user, args.package.name, build['build_no'])
```
Comment (dsludwig) [inline binstar_build_client/build_commands/submit.py]: `binstar.domain` is the api URL root (including the scheme) - so `https://api.anaconda.org` or `http://localhost:8080/api` - this gives a weird answer for where to go.

File: binstar_build_client/build_commands/submit.py [hunk 2]
```diff
@@ -95,17 +99,18 @@ def submit_build(binstar, args):
...
 
-def print_build_results(args, build):
+def print_build_results(args, build, binstar):
 
     log.info('')
     build_result_url = build.get('url')
     if not build_result_url:
-        build_result_url = 'http://anaconda.org/%s/%s/builds/matrix/%s' % (args.package.user, args.package.name, build['build_no'])
+        netloc = urlparse(binstar.domain).netloc
+        build_result_url = 'http://%s/%s/%s/builds/matrix/%s' % (netloc, args.package.user, args.package.name, build['build_no'])
```
Comment (dsludwig) [inline binstar_build_client/build_commands/submit.py]: netloc will be: `api.anaconda.org` for production. You should strip the api subdomain from the netloc if it exists. You might want to factor that out into a `get_anaconda_url()` helper.

Comment (PeterDSteinberg) [inline binstar_build_client/build_commands/submit.py]: Right.  Forgot about that.  I'll add the `get_anaconda_url` func.

**Comment by PeterDSteinberg** [comment]
The latest commit factors out the netloc modification into a helper function.

**Comment by dsludwig** [comment]
LGTM

**Pull request merged**